cypress/e2e/analytics/user-command-stats.cy.js

```diff
@@ -1,11 +1,11 @@
 suite(["@tier2"], "user commands stats", () => {
   it("are rendered", () => {
     cy.visit("/analytics/commands/user?limit=10000");
-    cy.get("#total-transferred").should("have.text", "4.861", {
+    cy.get("#total-transferred").should("have.text", "4.859", {
       timeout: 15000,
     });
-    cy.get("#total-fees").should("have.text", "377.994");
-    cy.get("#total-number-of-transactions").should("have.text", "27,858");
+    cy.get("#total-fees").should("have.text", "377.979");
+    cy.get("#total-number-of-transactions").should("have.text", "27,852");
```

Comment:
Are these only looking at canonical blocks now? It would be consistent since all values decreased.

Reply:
yes, only canonical